bundles/admin/admin-layereditor/view/AdminLayerForm/AdminLayerFormHandler.js

```diff
@@ -536,6 +536,13 @@ class UIHandler extends StateHandler {
         }
         // Take a copy
         const layer = { ...this.getState().layer };
+
+        // If layer options contain default values, remove them.
+        layer.options.hasOwnProperty('dynamicScreenSpaceError') && layer.options.dynamicScreenSpaceError === false && delete layer.options['dynamicScreenSpaceError'];
+        layer.options.hasOwnProperty('dynamicScreenSpaceErrorDensity') && layer.options.dynamicScreenSpaceErrorDensity === 0.00278 && delete layer.options['dynamicScreenSpaceErrorDensity'];
+        layer.options.hasOwnProperty('dynamicScreenSpaceErrorFactor') && layer.options.dynamicScreenSpaceErrorFactor === 4.0 && delete layer.options['dynamicScreenSpaceErrorFactor'];
+        layer.options.hasOwnProperty('dynamicScreenSpaceErrorHeightFalloff') && layer.options.dynamicScreenSpaceErrorHeightFalloff === 0.25 && delete layer.options['dynamicScreenSpaceErrorHeightFalloff'];
```

Comment:
Move this logic to the components instead. If the selected value is the default -> remove the key from the options. Otherwise add it to options. That way the default value is in a single location and weird bugs don't happen if it ever changes.